benchmark/sirun/Dockerfile

```diff
@@ -37,3 +37,14 @@ RUN mkdir -p /usr/local/nvm \
 	&& nvm install --no-progress 20.4.0 \
 	&& nvm alias default 18 \
 	&& nvm use 18
+
+RUN mkdir /opt/insecure-bank-js
+RUN git clone --depth 1 https://github.com/hdiv/insecure-bank-js.git /opt/insecure-bank-js
```

Comment:
Should we checkout a specific commit to prevent unexpected changes in the future? In this way we will need to do a manual change to update the application, and we are not going to have unexpected regressions from the external application.